src/test/lrsql/admin/protocol_test.clj

```diff
@@ -155,6 +183,49 @@
         (is (-> (adp/-ensure-account-oidc lrs username bad-issuer)
                 :result
                 (= :lrsql.admin/oidc-issuer-mismatch-error)))))
+    (testing "actor-delete"
+      (let [stmts [stmt-0 stmt-1 stmt-2 stmt-3]
+            ifis (->> (conj stmts (stmt-3 "object"))
+                      (map #(ua/actor->ifi (% "actor"))))
+            get-stmt-#s (fn []
+                          (reduce (fn [m actor-ifi]
+                                    (assoc m actor-ifi
+                                           (count (:statements (:statement-result (lrsp/-get-statements lrs auth-ident {:actor-ifi actor-ifi} []))))))
+                                  {} ifis))]
+        (lrsp/-store-statements lrs auth-ident stmts [])
+
+        (assert (every? #(> % 0) (vals (get-stmt-#s))))
```

Comment:
Why are these asserts rather than just `is`?

Reply:
I thought it might be worth it to have a separation between "is the test setting up correctly" and "is this passing the tests"

Reply:
That's an interesting idea, though I think letting the test output read everything out for you is probably better than wading through an assertion stack trace.

Reply:
I agree with @milt. Using only `is` instead of sprinkling in `assert` makes the tests more readable, plus in my view it tells a story of "these statements/actors/etc used to be here, now they're not."

Reply:
switched to use of `is`